keda/values.yaml

```diff
@@ -325,6 +325,7 @@ http:
   timeout: 3000
   keepAlive:
     enabled: true
+  minTlsVersion: TLS12
```

Comment:
Can you update the readme with the new setting please?

Reply:
Done!, I didn't see the comment yesterday sorry